### ISSUE TITLE
[Request] Temperature info

### ISSUE THREAD

**Issue by jonkristian** [piitaya/lovelace-climate-mode-entity-row#4]
This card is really nice! It might be out of the scope of this card, but it would be really cool to have some options for displaying temperatures like current / target.

**Comment by varadujagan** [comment]
yes, this will be a good idea, I'm also looking for this request.

**Comment by bkr1969** [comment]
Ditto

**Comment by lambtho12** [comment]
Hi, sorry to reopen this, but wouldn't it be possible to show the temperature as a "secondary_info?", just below the name?

Ideally we could have only three display options to keep stuff manageable:
- "{{current_temperature}}"
- "{{temperature}}"
- "{{current_temperature}} / {{temperature}}"
